pom.xml

```diff
@@ -42,11 +42,11 @@
     <version.io.opentracing>0.31.0</version.io.opentracing>
     <version.io.zipkin.brave-brave>4.6.0</version.io.zipkin.brave-brave>
     <version.io.zipkin.reporter-zipkin-sender-okhttp3>1.0.2</version.io.zipkin.reporter-zipkin-sender-okhttp3>
-    <version.junit>4.12</version.junit>
-    <version.org.assertj>3.8.0</version.org.assertj>
-    <version.org.testcontainers>1.12.0</version.org.testcontainers>
-    <version.com.squareup.okhttp3-okhttp>3.8.0</version.com.squareup.okhttp3-okhttp>
-    <version.org.awaitility-awaitility>3.0.0</version.org.awaitility-awaitility>
+    <version.junit>4.13.2</version.junit>
+    <version.org.assertj>3.22.0</version.org.assertj>
+    <version.org.testcontainers>1.12.5</version.org.testcontainers>
```

Comment:
this is not the latest version. Test containers uses jackson and the new version collides with spark. Therefore the testcontaienrs might get updated again once spark is updated.